src/legacy/core_plugins/kibana/public/discover/_index.scss

```diff
@@ -8,19 +8,19 @@
 //   monChart__legend--small
 //   monChart__legend-isLoading
 
-@import 'components/fetch_error/index';
-@import 'components/field_chooser/index';
-@import 'angular/directives/index';
-@import 'angular/doc_table/index';
+@import 'np_ready/components/fetch_error/index';
+@import 'np_ready/components/field_chooser/index';
+@import 'np_ready/angular/directives/index';
+@import 'np_ready/angular/doc_table/index';
 
 @import 'hacks';
 
 @import 'discover';
```

Comment:
These files and `_mixins.scss` should also be moved to `np_ready` folder

Reply:
Essentially this whole file should be moved to `np_ready` and then this file just imports the index file from there.

Reply:
That's a good point @cchaos , thanks for raising. I totally agree to have index files for the sub folders. But I would like to ignore the `np_ready` folder while doing this because it is just a temporary measure and will go away when we move discover to the new platform - so `_index.scss` for `components` and `angular`, but not for `np_ready`.

Reply:
The main reason to consider it for `np_ready` is if you have the manifest in `plugin/np_ready/_index.scss` like:

```
@import 'discover';
@import 'components/index';
...etc...
```

Then when you move/rename the whole `np_ready` folder, you don't have to change any of the imports in the `_index.scss` manifest file, you just change the single import **of** the index file.

Whereas right now you have many import paths directing to `/np_ready/...` explicitly that would need to change.

Reply:
You are right, I moved them into `np_ready` - doing it like this means the smallest change later when we migrate.